### ISSUE TITLE
Page "Session List" not found

### ISSUE THREAD

**Issue by megel** [Microsoft/AL#802]
An error is shown, when I try to extend the "Session List" page:

![grafik](https://user-images.githubusercontent.com/15800069/31757383-0afafa86-b4a9-11e7-8544-a93b131612bf.png)

My ```app.json``` contains:
```json
{
 "target": "Internal"
}
```

The page extension code:
![grafik](https://user-images.githubusercontent.com/15800069/31757315-c39be36c-b4a8-11e7-957d-8013a4eb5789.png)

**I'm not correct** (found out right now)
The page extension works, BUT VS Code / the VS Code Extension (Microsoft/AL) have the problem:
- VS Code is not able to open the Page "Session List"
- Page "Session List" is not accepted for
```javascript
pageextension 51000 "MyExt" extends "Session List"
{
}
```

The problem is still present after a restart of VS Code.

**Comment by StanislawStempin** [comment]
I cannot reproduce any problems with Session List. I can create a page extension for it, I can do F12 into Session List to see the metadata and I can set it as startupObjectId (page 9506) ID to launch the page from VS Code. 

Do you have all the right symbols referenced from app.json and present in the .alpackages cache folder?

**Comment by megel** [comment]
I have now also the problem after restart of my computer and after the restart of everything, that the following example NOT work and NOT compile:

```javascript
// Welcome to your new AL extension.
// Remember that object names and IDs should be unique across all extensions.
// AL snippets start with t*, like tpageext - give them a try and happy coding!

pageextension 50100 CustomerListExt extends "Customer List"
{
    trigger OnOpenPage();
    begin
        Message('App published: Hello world');
    end;
}
```
The output is:
```
Microsoft (R) AL Compiler version 0.9.13932
Copyright (C) Microsoft Corporation. All rights reserved

c:\Users\username\Documents\AL\ALProject6\HelloWorld.al(5,45): error AL0247: The target Page "Customer List" for the extension object is not found
Error: The package could not be created.
```

I started from scratch.
The app.json:
```json
{
  "id": "f3e00427-933f-4b19-887f-b21e1edbdc5d",
  "name": "ALProject6",
  "publisher": "Default publisher",
  "brief": "",
  "description": "",
  "version": "1.0.0.0",
  "compatibilityId": "1.0.0.0",
  "privacyStatement": "",
  "EULA": "",
  "help": "",
  "url": "",
  "logo": "",
  "capabilities": [],
  "dependencies": [],
  "screenshots": [],
  "platform": "11.0.0.0",
  "application": {
    "version": "11.0.0.0",
    "locale": "W1"
  },
  "idRange": {
    "from": 50100,
    "to": 50149
  },
  "target": "Internal"
}```

**Comment by megel** [comment]
Was there an update on any component VS Code / MS in the last hours?

**Comment by StanislawStempin** [comment]
There were no updates. Are you getting errors in VS Code Error List window as well or only if you do Ctrl + Shift + B?

**Comment by megel** [comment]
I belive I found the problem:

This version don't work:
![grafik](https://user-images.githubusercontent.com/15800069/31770661-3caf812a-b4d9-11e7-8ac3-9aebbee4e179.png)

This elder version works:
![grafik](https://user-images.githubusercontent.com/15800069/31770706-67427c62-b4d9-11e7-966f-58608d5cac7a.png)

**Comment by StanislawStempin** [comment]
Are you using the VSIX version that corresponds to that symbol file? You should be using AL Language 0.10.13928

**Comment by megel** [comment]
Sorry for the late reply, I use the following vsix from AppSource / AppStore:
![grafik](https://user-images.githubusercontent.com/15800069/31778023-dc1203bc-b4ef-11e7-9a27-e38c9ee6492c.png)
The extension from developer preview (docker image) is elder.

**Comment by StanislawStempin** [comment]
Try installing again if it's still not present after restart. Any recent version of VS Code should work fine with our vsix.

**Comment by megel** [comment]
Sorry, no way to install - also not after restart :(

Do you have a copy of the installed extension?

**Comment by StanislawStempin** [comment]
Could you try deleting all AL related folders under  user/<yourUserName>/.vscode/extensions/ and try to reinstall?

**Comment by megel** [comment]
I be live I have checked this before - but tomorrow I will try again with a clean image.

**Comment by megel** [comment]
So, a new day a new try and I found my mistake!

You switched the Icon, description & the maybe the IDs of the extension. So I didn't realized that both were installed at the same time. I saw all actions twice and unfortunately I disabled the wrong vsix.

Marketplace:
![grafik](https://user-images.githubusercontent.com/15800069/31807364-084334c8-b56e-11e7-8413-b24906c5bfaf.png)

Dev-Preview:
![grafik](https://user-images.githubusercontent.com/15800069/31807361-fef37c20-b56d-11e7-9100-b4015370359a.png)

After a short retest - everything works as expected:
![grafik](https://user-images.githubusercontent.com/15800069/31807569-388ef35a-b56f-11e7-8564-f858a7017630.png)

Problem found & solved!
